pennylane/ops/cv.py

```diff
@@ -32,6 +32,8 @@
    :math:`(\hat{\mathbb{1}}, \hat{x}, \hat{p})` for single modes
    and :math:`(\hat{\mathbb{1}}, \hat{x}_1, \hat{p}_2, \hat{x}_1,\hat{p}_2)` for two modes .
 """
+# pylint: disable=abstract-method
```

Comment:
What was the reason for this lint comment? It seems like an odd one to disable

Reply:
The following is obtained for every class in `pennylane/ops/cv.py`:
```python
pennylane/ops/cv.py:732:0: W0223: Method 'decomposition' is abstract in class 'Operation' but is not overridden (abstract-method)
pennylane/ops/cv.py:769:0: W0223: Method '_matrix' is abstract in class 'Operator' but is not overridden (abstract-method)
pennylane/ops/cv.py:769:0: W0223: Method 'diagonalizing_gates' is abstract in class 'Observable' but is not overridden (abstract-method)
```

Reply:
Oh :( This is because we've added Qubit-specific methods over the years. Might be worth adding a comment to explain this is the reason it is disabled.